src/Process/GlobeTileProcessing.js

```diff
@@ -64,7 +64,7 @@ function frustumCullingOBB(node, camera) {
 }
 
 export function globeCulling(node, camera) {
-    return !(frustumCullingOBB(node, camera) && horizonCulling(node));
+    return !(frustumCullingOBB(node, camera) && (node.level < 2 || horizonCulling(node)));
```

Comment:
Replace 2 by process's parameter.
`globeCulling` with `node.level < 2 ` can be important with a different `scheme tile`